app/models/spree/avalara_transaction.rb

```diff
@@ -29,7 +29,17 @@ def commit_avatax(invoice_dt = nil, return_auth = nil)
 
     def commit_avatax_final(invoice_dt = nil, return_auth = nil)
       if document_committing_enabled?
-        if tax_calculation_enabled?
+        # Only commit
+        #   (if gem spree preference is set to true)
+        #   AND
+        #   (
+        #     if the order was never handled by the FullscriptTransactionCalculator)
+        #     OR
+        #     the order tax was successfuly calculated by avata
+        #   )
+        #
+        #  This applies to both orders and returns.
+        if tax_calculation_enabled? && (order.tax_application.blank? || order.tax_application == 'avatax')
```

Comment:
And basically store would be the value on those other ones?

Reply:
'store' and 'none' are the other 2 values that could be set

Reply:
makes sense to me